minigu-test/Cargo.toml

```diff
@@ -5,9 +5,16 @@ name = "minigu-test"
 repository.workspace = true
 version.workspace = true
 
+[dependencies]
+glob = "0.3"
+libtest-mimic = "0.8.1"
+minigu = { workspace = true }
+sqllogictest = { workspace = true }
+
```

Comment:
Versioned dependencies should be declared in `Cargo.toml` in the workspace root.